csharp/Ice/Context/README.md

```diff
@@ -0,0 +1,36 @@
+# Context
+
+The Context demo shows the 3 different ways to set a request context in a client.
+
+A request context is a `dictionary<string, string>` carried by all requests. It is empty by default, and the
+application is free to set any entry in this dictionary.
+
+> [!NOTE]
+> The Ice protocol does not support response contexts, only request contexts.
+
+You can build the client and server applications with:
+
+``` shell
+dotnet build
+```
+
+First start the Server program:
+
+```shell
+cd Server
+dotnet run
+```
+
+or
```

Comment:
Do we provide ServerAMD we the Context demo?

Reply:
No, will fix.